other/install_2-1_mysql.sql

```diff
@@ -807,9 +807,8 @@ CREATE TABLE {$db_prefix}messages (
 	approved TINYINT NOT NULL DEFAULT '1',
 	likes SMALLINT UNSIGNED NOT NULL DEFAULT '0',
 	PRIMARY KEY (id_msg),
-	UNIQUE idx_id_board (id_board, id_msg),
+	UNIQUE idx_id_board (id_board, id_msg, approved),
 	UNIQUE idx_id_member (id_member, id_msg),
-	INDEX idx_approved (approved),
```

Comment:
Why removing this Index?

Reply:
It is unused.  In general, it is a bad idea to put a standalone index on a boolean.  Low cardinality.  

However, moving it over to the id_board index helps *A LOT*.

Reply:
the problem is here, that you change the defintion of the unqiue index, so a post/message can exsits now more times in the same board.

Reply:
(I think you cross posted into the wrong thread, but I'll respond to this concern here, because it is a good one.)

Both the board_id and the approved flag are on the message, ie., dependent on id_message.  So that cannot happen.